src/components/toasts/OfflineToast.js

```diff
@@ -1,15 +1,15 @@
 import lang from 'i18n-js';
 import React from 'react';
-import { web3Provider } from '../../handlers/web3';
 import Toast from './Toast';
 import { useAccountSettings, useInternetStatus } from '@/hooks';
-import { Network } from '@/chains/types';
+import { ChainId } from '@/chains/types';
+import { useConnectedToHardhatStore } from '@/state/connectedToHardhat';
 
 const OfflineToast = () => {
   const isConnected = useInternetStatus();
-  const { network } = useAccountSettings();
-  const providerUrl = web3Provider?.connection?.url;
-  const isMainnet = network === Network.mainnet && !providerUrl?.startsWith('http://');
```

Comment:
the only reason the `web3Provider` was used here was to check the provider URL to see if it's the hardhat URL; moved it over to using the new `connectedToHardhat` instead